profile/merge.go

```diff
@@ -85,6 +86,55 @@ func Merge(srcs []*Profile) (*Profile, error) {
 	return p, nil
 }
 
+// Normalize normalizes the source profiles by multiplying each value in profiles by the
+// absolute value of the ratio of the sum of the base profile value's of that sample type
+// to the sum of the source profiles value of that sample type.
+func Normalize(srcProfiles, baseProfiles []*Profile) error {
```

Comment:
I think this function should merely take two profiles. The caller should do the combining. See another comment I left.